test_scripts/API/VehicleData/StabilityControlsStatus/020_UnsubscribeVehicleData_StabilityControlsStatus_partially_disallowed_by_policy.lua

```diff
@@ -0,0 +1,44 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0253-New-vehicle-data-StabilityControlsStatus.md
+--
+-- Description: UnsubscribeVehicleData RPC with `stabilityControlsStatus` parameter 
+-- which is NOT allowed by Policies and other parameters which is allowed by Policies
+--
+-- Precondition:
+-- 1) SDL and HMI are started
+-- 2) App is registered
+-- 3) PTU is successfully performed
+-- 4) App is activated
+-- 5) App is subscribed on gps vehicle data
+--
+-- Steps:
+-- 1) App send UnsubscribeVehicleData (with stabilityControlsStatus = true and gps = true) request to SDL
+-- SDL does:
+--  - cut stabilityControlsStatus off
+--  - send VehicleInfo.UnsubscribeVehicleData request with (gps = true) to HMI
```

Comment:
@LitvinenkoIra sends

Reply:
@AByzhynar Done in 4689ec2

Reply:
@LitvinenkoIra This is wrong. Please get the idea from this comment: https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/15/files#r425053019